packages/auth-construct/src/construct.test.ts

```diff
@@ -967,6 +967,39 @@ void describe('Auth construct', () => {
       });
     });
 
+    void it('disables implicit grant OAUTH flow by default', () => {
+      const app = new App();
+      const stack = new Stack(app);
+      new AmplifyAuth(stack, 'test');
+      const template = Template.fromStack(stack);
+      template.hasResourceProperties('AWS::Cognito::UserPoolClient', {
+        AllowedOAuthFlows: ['code'],
+      });
```

Comment:
wouldn't we want to assert the absence of implicit grant or presence of `implicitCodeGrant: false`? This test would perhaps pass without this change as well.

Reply:
In this case, we really are asserting that 'code' is the only allowed OAuth flow in the UserPoolClient, since that is the only other available option. 
I can rename the test description for clarity

Reply:
updated